expat/xmlwf/xmlwf.c

```diff
@@ -1175,8 +1175,9 @@ tmain(int argc, XML_Char **argv) {
       if (! userData.fp) {
         tperror(outName);
         exitCode = XMLWF_EXIT_OUTPUT_ERROR;
+        free(outName);
```

Comment:
Heh, pretty obvious when you look at it, there was no reason to have the free only in the -k case.

Reply:
@timbray thanks for having a look!

I noticed now that the call to `cleanupUserData(&userData)` should either be moved out of the conditional or be dropped altogether. Keeping and moving seems more robust towards the future, but either approach seems to be valid. What do you think?

CC @hwt0415

Reply:
I thought about this again. There is no chance that we have information about the doctype or notations (derived from from file content) this early in the process for each file. I have dropped the call to `cleanupUserData(&userData)` as a result.